bounties/packagist/hotarucms/1/README.md

```diff
@@ -0,0 +1,3 @@
+# Overview
+
+hotarucms/hotarucms is a Bookmarking CMS for PHP 5.3, this package is vulnerable to Cross-site Scripting (XSS) via. the `admin_index.php?page=settings SITE_NAME` field.
```

Comment:
I wasn't able to directly verify because I had some problems with the installation, however it seems nor PR/commits have been accepted/released since 2015, so should be still valid (note the vulnerable version 1.7.2 is the actual one).

Reply:
No worries, I will leave this one up for now also and see if any other Sheriffs have any luck validating this.

Cheers 😎

Reply:
This vulnerability is valid! :+1: